chaincfg/params.go

```diff
@@ -354,11 +354,11 @@ type Params struct {
 	// Deprecated: This will be removed in the next major version bump.
 	BlockTaxProportion uint16
 
-	// Checkpoints ordered from oldest to newest.
+	// Checkpoints previously housed manual checkpoint ordered from oldest to
+	// newest. It is no longer used in any way since manual checkpoints no
+	// longer exist.
 	//
-	// Note: Only the most recent checkpoint is used and all others are ignored.
-	// This is left as a slice to avoid a major version bump for the module and
-	// will likely be changed to only allow a single checkpoint in the future.
+	// Deprecated: This will be removed in the next major version bump.
 	Checkpoints []Checkpoint
```

Comment:
Do you think it would be worth commenting on the `AssumeValid` param to indicate that in addition to skipping some validation checks, it is now also used to determine the old forks checkpoint?

Reply:
Good point.  I added the following blurb:

> It is also used to determine the old forks rejection checkpoint.